src/app/public/modules/character-counter/character-counter-indicator.component.html

```diff
@@ -0,0 +1,7 @@
+
+<label
```

Comment:
Does this need to be an aria live region to announce the updated count on screen readers?

Reply:
@Blackbaud-ToddRoberts I added an aria-live span to this that puts the character count in sentence form--just having it read in the label would result in "John. Four ten." instead of something like "four characters out of ten". The aria-live is set to polite, which means it waits for a pause to announce the most recent value. Let me know if that works, if not I can try to add some debouncing.

Reply:
This approach seems good to me. UK gov't design system character count works the same way and I imagine their requirements are pretty strict.